### ISSUE TITLE
Blocks braitnreepayments javascripts!

### ISSUE THREAD

**Issue by syedhassan** [EFForg/privacybadgerchrome#742]
Hi,

As a indie-developer, I use braintree payments and I see that privacy badger disallows any javascripts to load from braintree payments.

**Comment by syedhassan** [comment]
FYI @braintreeps

**Comment by crookedneighbor** [comment]
@syedhassan I'm a developer at Braintree. Thanks for bringing this up. We are monitoring this issue and discussing internally, and will help if possible.

**Comment by cooperq** [comment]
I suspect that cookies are being set with a braintree domain, which would end up blocking braintree. Can you tell me exactly which domains privacy badger is blocking?

**Comment by syedhassan** [comment]
@cooperq attaching a screenshot (though I moved the button to right to enable them).

The domains that were getting blocked were 

api.braintreegateway.com
js.braintreegateway.com

cc @crookedneighbor 

[http://i.imgur.com/XSQKjY8.png](Screenshot)

**Comment by crookedneighbor** [comment]
@cooperq Any update on this?

**Comment by cooperq** [comment]
@crookedneighbor if {api | js}.braintreegateway.com were blocked from getting / setting third party cookies, as oppposed to being blocked entirely, would braintree still work? 
If one of you can give me a domain which uses braintree to test this on, that would be excellent.

**Comment by crookedneighbor** [comment]
We have a demo here: https://developers.braintreepayments.com/start/hello-client/javascript/v2#demo

**Comment by cooperq** [comment]
Ah, interesting it looks like the cookie gets set when someone visits braintreepayments.com directly. But someone who is just encountering braintreepayments as a third party js include will, ironically, never have this problem! Basically this will only affect developers who are using the braintree api. Regardless I have added braintreegateway.com to the cookie block list. Which should fix the problem for everyone. 

Fixed by f1b424f
